packages/styles/stories/Components/Forms/form-message/form-message.stories.js

```diff
@@ -26,7 +26,7 @@ Read-only&nbsp;&nbsp;&nbsp; | \`is-readonly\`&nbsp;&nbsp;&nbsp; | Used to displa
     }
 };
 
-export const Success = () => `<div class="fd-form-item">
+export const Success = () => `<div class="fd-form-item" style="min-height: 100px;">
```

Comment:
I prefer to fix the `iframeHeight` instead of doing this, here is the ticket https://github.com/SAP/fundamental-styles/issues/4259